webdriver/tests/bidi/browsing_context/print/background.py

```diff
@@ -16,8 +13,8 @@
 </style>
 """
 
-BLACK_DOT_PNG = "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVQIW2NgYGD4DwABBAEAwS2OUAAAAABJRU5ErkJggg=="
```

Comment:
Instead of increasing the size of the image (note it's still named as black dot) lets add an element to the test page which eg. is 100x100 in size.

Reply:
The only change here was to increase the page size, and to update the base64 to match the new result.

What do you mean by adding a 100x100 element? The page background is painted to a specific color (in this case, black). I don't see how adding an element would improve the test.

I agree with the variable name change observation though. Will update it.

Reply:
Done.